src/db.rs

```diff
@@ -1162,6 +1162,10 @@ impl DB {
         DBRawIterator::new(self, &opts)
     }
 
+    pub fn raw_iterator_opt(&self, opts: ReadOptions) -> DBRawIterator {
```

Comment:
Why do you pass `ReadOptions` by value ?

```suggestion
    pub fn raw_iterator_opt(&self, opts: &ReadOptions) -> DBRawIterator {
```

Reply:
Oh, good catch!